src/features/threads/DisplayTempUb3rStatus.js

```diff
@@ -0,0 +1,27 @@
+const Feature = require("../../core/Feature");
+const Threads = require("../../sections/Threads");
+
+class DisplayTempUb3rStatus extends Feature {
+  constructor() {
+    super({
+      section: Threads,
+      name: "Display Temp Ub3r Status",
+      default: true,
```

Comment:
Do you mind if we either disable this by default or make it obvious that its from HFX?
Without specifying that this new text is from HFX, I don't want users who just installed seeing this and making a thread about it without realizing it is from the extension.

Reply:
disabled by default